### ISSUE TITLE
Feature Request: Idle Indicator to also show Bootstrapping and Running

### ISSUE THREAD

**Issue by rickymac68** [input-output-hk/cardano-rt-view#135]
Request the node Idle indicator to also show when the node is Bootstrapping and Running. That way it would indicate 3 states:
Idle
Bootstrapping
Running

**Comment by denisshevchenko** [comment]
What is `Bootstrapping` in this case? Similar to `Syncing`?

**Comment by rickymac68** [comment]
@denisshevchenko Yes, Bootstrapping is the same as Syncing, or very similar to Syncing. It's when the operator first starts the node and node is not fully operational yet... currently about 2 to 3 minutes on 1.24.2

What would be useful to have is that where the Idle indicator is showing and is red in color, the Idle indicator stays there and turns colors:
Idle = Red
Bootstrapping =  Yellow
Running = Green
